src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheckTest.java

```diff
@@ -65,20 +65,70 @@ public void testDefault() throws Exception {
+        configurationTestCast.addAttribute("tokens", "TYPECAST");
         final String[] expected = {
             "88:21: " + getCheckMessage(MSG_WS_TYPECAST),
         };
-        verify(checkConfig, getPath("InputWhitespace.java"), expected);
+        verify(configurationTestCast, getPath("InputWhitespace.java"), expected);
     }
 
     @Test
     public void testSemi() throws Exception {
+        final DefaultConfiguration configurationTestSemi =
+                createCheckConfig(WhitespaceAfterCheck.class);
+        configurationTestSemi.addAttribute("tokens", "SEMI");
         final String[] expected = {
             "54:23: " + getCheckMessage(MSG_WS_NOT_FOLLOWED, ";"),
             "54:29: " + getCheckMessage(MSG_WS_NOT_FOLLOWED, ";"),
             "103:19: " + getCheckMessage(MSG_WS_NOT_FOLLOWED, ";"),
         };
-        verify(checkConfig, getPath("InputBraces.java"), expected);
+        verify(configurationTestSemi, getPath("InputBraces.java"), expected);
+    }
+
+    @Test
+    public void testLiteralWhile() throws Exception {
+        final DefaultConfiguration configurationTestLiteralWhile =
+                createCheckConfig(WhitespaceAfterCheck.class);
+        configurationTestLiteralWhile.addAttribute("tokens", "LITERAL_WHILE");
+        final String[] expected = {
+            "39:14: " + getCheckMessage(MSG_WS_NOT_FOLLOWED, "while"),
+        };
+        verify(configurationTestLiteralWhile, getPath("InputWhitespaceAfter.java"), expected);
+    }
+
+    @Test
+    public void testLiteralIf() throws Exception {
+        final DefaultConfiguration configurationTestLiteralIf =
+                createCheckConfig(WhitespaceAfterCheck.class);
+        configurationTestLiteralIf.addAttribute("tokens", "LITERAL_IF");
+        final String[] expected = {
+            "18:11: " + getCheckMessage(MSG_WS_NOT_FOLLOWED, "if"),
+        };
+        verify(configurationTestLiteralIf, getPath("InputWhitespaceAfter.java"), expected);
+    }
+
+    @Test
+    public void testLiteralElse() throws Exception {
+        final DefaultConfiguration configurationTestLiteralElse =
+                createCheckConfig(WhitespaceAfterCheck.class);
+        configurationTestLiteralElse.addAttribute("tokens", "LITERAL_ELSE");
+        final String[] expected = {
+            "27:15: " + getCheckMessage(MSG_WS_NOT_FOLLOWED, "else"),
+        };
+        verify(configurationTestLiteralElse, getPath("InputWhitespaceAfter.java"), expected);
+    }
+
+    @Test
+    public void testLiteralFor() throws Exception {
+        final DefaultConfiguration configurationTestLiteralFor =
+                createCheckConfig(WhitespaceAfterCheck.class);
+        configurationTestLiteralFor.addAttribute("tokens", "LITERAL_FOR");
+        final String[] expected = {
+            "51:12: " + getCheckMessage(MSG_WS_NOT_FOLLOWED, "for"),
+        };
+        verify(configurationTestLiteralFor, getPath("InputWhitespaceAfter.java"), expected);
```

Comment:
I see tests for `while`, `if`, `else`, `for` but not for `do`. Can you please add something for `do`.